manifests/s/Saladict/app/3.0.6/Saladict.app.yaml

```
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: Saladict.app
+PackageVersion: 3.0.6
+DefaultLocale: en-US
+ManifestType: version
+ManifestVersion: 1.5.0
```

```suggestion
ManifestVersion: 1.9.0
```

Version of manifest are outdated.
清单版本已过时